### ISSUE TITLE
display issue with Peptide Ratio Normalization

### ISSUE THREAD

**Issue by jcanderan** [compomics/reporter#19]
This is a duplicate of the same issue with Peptideshaker (that was fixed).

When doing Peptide Ratio Normalization, you'll see the following (some of it is truncated because it went off the screen to the side):

```
Peptide Ratio Normalization. Please Wait...
110% 120% 130% 140% 150% 160% 170% 180% 190% 200% 210% 220% 230% 240% 250% 260% 270% 280% 290% 300% 310% 320% 330% 340% 350% 360% 370% 380% 390% 400% 410% 420% 430% 440% 450% 460% 470% 480% 490%>
1010%
1020%
1030%
1040%
1050%
1060%
1070%
1080%
1090%
1100%
1110%
1120%
```
(it continues on for quite a while).

Thanks for fixing all the issues with reporter. I appreciate it!

**Comment by hbarsnes** [comment]
Ah, yes, I noticed this myself as well. I will try to look into it when I get the chance.

**Comment by hbarsnes** [comment]
I just released Reporter v0.9.4 which should correct the issue with the progress display. If this is not the case, please let me know and I'll reopen the issue.
